develop/parachain-devs/parachain-launch/chain-spec.md

```diff
@@ -0,0 +1,113 @@
+
+Note that you can customize outer node settings after genesis. However, nodes only add peers that use the same [`protocolId`](https://paritytech.github.io/polkadot-sdk/master/polkadot_parachain_lib/chain_spec/trait.ChainSpec.html#tymethod.protocol_id){target=_blank}.
+
+## Customizing the Genesis Configuration
+
+All nodes in the network must agree on the genesis state before they can agree on any subsequent blocks. The information configured in the genesis portion of a chain specification is used to create a genesis block. It takes effect when you start the first node and cannot be overridden with command-line options. However, you can configure some information in the genesis portion of a chain specification. For example, you can customize the genesis portion of the chain specification to include information such as:
+
+- Initial account balances
+- Accounts that are initially part of a governance council
+- The account that controls the `sudo` key
+- Any other genesis state for a pallet
+
+Nodes also include the compiled WebAssembly for the runtime logic on the chain, so the initial runtime must also be supplied in the chain specification.
+
+## Declaring storage items for a runtime
+
+In most cases, a runtime requires some storage items to be configured at genesis. This includes the initial state for pallets, for example, how much balance `Alice` or `Bob` exists, or the account that has sudo permissions.
+
+These storage values are configured in the genesis portion of the chain specification. You can either create a [patch](https://paritytech.github.io/polkadot-sdk/master/sc_chain_spec/index.html#chain-spec-formats){target=_blank}  file for [`chain-spec-builder`](https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/reference_docs/chain_spec_genesis/index.html){target=_blank} to ingest, or if you're using `chain_spec.rs`, the pallet can be configured in the code.
+
+<!-- TODO: Add a page for genesis configuration? -->
+
+## Creating a Custom Chain Specification
+
+There are two methods for creating a chain specification: 
+
+1. Utilizing `polkadot-parachain` or compiled node binary
+2. Utilizing `chain-spec-builder`
+
+If you are creating a one-off network for development, testing, or demonstration purposes, you might want a fully customized chain specification.
+To create a completely customized chain spec, you can export the default chain spec to JSON format, then edit the fields in the JSON file.
+For example, you can use the `build-spec` sub-command to export the chain specification to a JSON file:
+
+### Using Node Binary
+
+
+!!!warning "Be aware of the Omninode"
+    As mentioned, there are two methods for generating a chain specification. The preferred method is [`chain-spec-builder`](https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/reference_docs/chain_spec_genesis/index.html){target=_blank}, however the `build-spec` method can be utilized as well.
```

Comment:
The link to chain-spec-builder is: https://paritytech.github.io/polkadot-sdk/master/staging_chain_spec_builder/index.html

Reply:
And also why mention chain-spec-builder in `Using node binary` section? I'd remove it...